common/lib/xmodule/xmodule/course_module.py

```diff
@@ -701,16 +701,22 @@ class CourseFields(object):
 
     teams_configuration = Dict(
         display_name=_("Teams Configuration"),
+        # Translators: please don't translate "id".
         help=_(
             'Specify the maximum team size and topics for teams inside the provided set of curly braces. '
             'Make sure that you enclose all of the sets of topic values within a set of square brackets, '
             'with a comma after the closing curly brace for each topic, and another comma after the '
             'closing square brackets. '
             'For example, to specify that teams should have a maximum of 5 participants and provide a list of '
-            '2 topics, enter the configuration in this format: '
-            '{"topics": [{"name": "Topic1Name", "description": "Topic1Description", "id": "Topic1ID"}, '
-            '{"name": "Topic2Name", "description": "Topic2Description", "id": "Topic2ID"}], "max_team_size": 5}. '
+            '2 topics, enter the configuration in this format: {example_format}. '
             'In "id" values, the only supported special characters are underscore, hyphen, and period.'
+        ).format(
+            # Put the sample JSON into a format variable so that translators
+            # don't muck with it.
+            example_format=(
```

Comment:
This change looks great. Would it work to actually use a dict here, rather than escaping it into a string. That would make it easier to read, and make it more likely that the JSON is valid.